### PR TITLE
[BD-46] feat: refactoring design tokens names

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

- rename design tokens according to the decision in the ticket above
- add missed tokens

image
image

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [ ] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [ ] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [ ] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tokens/src/alias/color.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
         "gray": { "value": "{color.gray.100.value}" }
       },
       "border": {
+        "base": { "value": "{color.gray.200.value}", "source": "$border-color" },
```
Comment (viktorrusakov) [inline tokens/src/alias/color.json:47]: this should be outside of `theme` key

File: tokens/src/components/Modal.json [hunk 2]
```diff
@@ -13,6 +13,7 @@
       }
     },
     "content": {
+      "color": { "value": "null", "source": "$modal-content-color" },
```
Comment (viktorrusakov) [inline tokens/src/components/Modal.json:16]: please check that variables in this file are present in `_variables.scss` file in `Modal` component, I think some of them were deleted in one of the previous commits.

File: tokens/src/alias/color.json (hunk 1 above)
Comment (PKulkoRaccoonGang) [reply tokens/src/alias/color.json:47]: moved thanks

File: tokens/src/components/Form/other.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
+{
+  "other": {
+    "form": {
+      "control": {
+        "cursor": { "value": "null", "source": "$custom-control-cursor" },
+        "range": {
+          "track": {
+            "cursor": { "value": "pointer", "source": "$custom-range-track-cursor" }
```
Comment (viktorrusakov) [inline tokens/src/components/Form/other.json:8]: I think it would be better to have `"range-track-cursor"` as a single key instead of this nesting (since there are no more variables here related to `track` or `cursor` keys)

File: tokens/src/components/Form/size.json [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+          },
+          "border": { "value": "calc({size.form.input.width.border.value} * 2)", "source": "$input-height-border" }
+        },
+        "width": {
+          "hover": { "value": "0.063rem", "source": "$input-hover-width" },
+          "focus": { "value": "0.063rem", "source": "$input-focus-width" },
+          "border": { "value": "{input.btn.border.width.value}", "source": "$input-border-width" }
+        },
+        "border": {
+          "radius": {
```
Comment (viktorrusakov) [inline tokens/src/components/Form/size.json:40]: should the order of keys be inverted here? i.e. first `radius` and then `border`, like above with `height` and `width` attributes, they have `border` inside them instead of outside

File: tokens/src/components/Form/typography.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+          "base": { "value": "{input.btn.line.height.base.value}", "source": "$input-line-height" },
+          "sm": { "value": "{input.btn.line.height.sm.value}", "source": "$input-line-height-sm" },
+          "lg": { "value": "{input.btn.line.height.lg.value}", "source": "$input-line-height-lg" }
+        }
+      },
+      "control": {
+        "label": {
+          "floating": {
+            "text": { 
+              "value": "{color.form.input.bg.base.value}",
```
Comment (viktorrusakov) [inline tokens/src/components/Form/typography.json:24]: this should probably be in `color` category

File: tokens/src/components/Form/typography.json [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+      "feedback": {
+        "font": {
+          "size": { "value": "{font.size.small.base.value}", "source": "$form-feedback-font-size" }
+        },
+        "tooltip": {
+          "font": {
+            "size": { "value": "{font.size.sm.value}", "source": "$form-feedback-tooltip-font-size" }
+          },
+          "line-height": { "value": "{line-height.base.value}", "source": "$form-feedback-tooltip-line-height" },
+          "opacity": { "value": ".9", "source": "$form-feedback-tooltip-opacity" }
```
Comment (viktorrusakov) [inline tokens/src/components/Form/typography.json:61]: this is also not related to `typography` I think, maybe better to move it to `other` (or `color`)?

File: tokens/src/components/Form/size.json [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+          "radio": {
+            "indicator": {
+              "radius": { "value": "50%", "source": "$custom-radio-indicator-border-radius" }
+            }
+          }
+        }
+      },
+      "select": {
+        "icon": {
+          "padding": { "value": ".5625rem", "source": "$select-icon-padding" }
```
Comment (viktorrusakov) [inline tokens/src/components/Form/size.json:136]: this is probably a `spacing` token

File: tokens/src/components/Form/elevation.json [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+          "focus": { "value": "{elevation.form.input.focus.value}", "source": "$custom-file-focus-box-shadow" }
+        },
+        "select": {
+          "border": {
+            "base": { "value": "inset 0 1px 2px rgba(0, 0, 0, .075)", "source": "$custom-select-box-shadow" },
+            "focus": { "value": "{input.btn.focus.box-shadow.value}", "source": "$custom-select-focus-box-shadow" }
+          }
+        }
+      },
+      "tooltip": {
```
[the remark this answers is not in the document]
Comment (PKulkoRaccoonGang) [reply tokens/src/components/Form/elevation.json:43]: Ok, moved

File: tokens/src/components/Form/elevation.json [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+            "indeterminate": { "value": "none", "source": "$custom-checkbox-indicator-indeterminate-box-shadow" }
+          }
+        },
+        "range": {
+          "track": { "value": "inset 0 .25rem .25rem rgba(0, 0, 0, .1)", "source": "$custom-range-track-box-shadow" },
+          "thumb": {
+            "base": { "value": "0 .1rem .25rem rgba(0, 0, 0, .1)", "source": "$custom-range-thumb-box-shadow" },
+            "focus": {
+              "base": { "value": "0 0 0 1px {body.bg.value}, {size.form.input.width.focus.value}", "source": "$custom-range-thumb-focus-box-shadow" },
+              "width": { "value": "{size.form.input.width.focus.value}", "source": "$custom-range-thumb-focus-box-shadow-width" }
```
[the remark this answers is not in the document]
Comment (PKulkoRaccoonGang) [reply tokens/src/components/Form/elevation.json:28]: Yeap, thanks

File: tokens/src/components/Form/other.json (hunk 3 above)
Comment (PKulkoRaccoonGang) [reply tokens/src/components/Form/other.json:8]: Yes, your right

File: tokens/src/components/Form/typography.json (hunk 5 above)
Comment (PKulkoRaccoonGang) [reply tokens/src/components/Form/typography.json:24]: done, thx

File: tokens/src/components/Form/size.json (hunk 7 above)
Comment (PKulkoRaccoonGang) [reply tokens/src/components/Form/size.json:136]: Moved, thanks

File: tokens/src/components/Form/typography.json (hunk 6 above)
Comment (PKulkoRaccoonGang) [reply tokens/src/components/Form/typography.json:61]: Yes, ok
thanks

**Review by viktorrusakov — APPROVED** [review]
LGTM!

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/1830&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2022-12-08+19:44&entry.1917517419=2022-12-15+10:13&entry.2133058324=Yes) so we can improve your experience in the future.
